src/components/introduction.js

```diff
@@ -4,28 +4,26 @@ import { Link } from 'react-router-dom'
 class Introduction extends React.PureComponent {
   render() {
     return (
-      <section>
```

Comment:
This has been moved to a div for styling to be consistent across page-level components